### PR TITLE
Turn dnssec-oracle into npm module

### PR DESCRIPTION
**Pull request by makoto**
Followed [Truffle npm package procedure](http://truffleframework.com/docs/getting_started/packages-npm). This package allows you to include smart contract as well as accessing abi from frontend.

This is my first attempt to create a npm package so feel free to point me out if there are certain things missing.

NOTE: I cannot publish to npm for another 24 hours as I unpublished previous package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "name": "dnssec-oracle",
```
Comment (decanus) [inline package.json]: scope to ens domains. So ```@ensdomains/dnssec-oracle```

File: README.md [hunk 2]
```diff
@@ -32,6 +32,27 @@ To run linting, use solium:
 solium --dir ./contracts
 ```
 
+## Including DNSSEC Oracle in your project
+
+### Installation
+
+```
+npm install dnssec-oracle
```
Comment (Arachnid) [inline README.md]: Maybe add `--save`.

File: package.json (hunk 1 above)
Comment (makoto) [reply package.json]: Just a generic question. Currently `dnssec-oracle` is owned by my account. How do we allow anyone from our team to publish under `@ensdomains/dnssec-oracle`?

Comment (decanus) [reply package.json]: @makoto I will need to add you as an admin to the ensdomains npm organization and then you can publish it public

File: .npmignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```
Comment (jefflau) [inline .npmignore]: You should be committing this to source control

**Pull request merged**